submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+* **Minimize fluid entity usage, particularly pumps** - In an optimized setup, the fluid system accounts for less than 1% of UPS drain. On the other hand, a setup with pressurized steam pipes (one pump per two exchangers) had an extra UPS overhead of 15%. Hence it is preferable to use very few pumps, while afterwards one may consider removing storage tanks (starting from steam storage).
+
+* **Prioritize optimizing what takes up the most time** - In Factorio, the heat manager, the fluid manager, and the electric network are simulated in parallel. Therefore, the slowest of these three factors causes the UPS bottleneck for the group while the other two have little to no impact. You can check this by opening the debug settings with the `F4` key and selecting "show-time-usage" and looking at which factor lists the largest numbers for update times. Based on this you can determine what to add or remove from your designs. For example, if the fluid manager costs comparatively little time, you can add more steam storage without worry.
+
+* **If you want to build a megabase, transition to solar power eventually** - In the end nuclear cannot compete with solar, which has simpler calculations and excellent performance optimizations. However, since solar power is more expensive than nuclear power in terms of setup costs and space usage, starting your large scale power production with a few GW of nuclear energy is still worth the trouble since the impact at that scale is small.
+
+Given these guidelines, you'll notice that the designs featured in earlier images are not the most UPS friendly. They don't have to be, unless you are going to build several copies of them. Nevertheless, lets see how the design changes when we apply these guidelines to an example.
+
+<!--VISUAL 6A/6B (slider comparison): To optimize for UPS we have removed non-essential heat pipes and steam storage. We also reduced the steam turbine count to fit the 1.718 ratio with heat exchangers. -->
+
+### Conclusion
+
+After going through the nuts and bolts nuclear power, hopefully it now seems easier to get into, more practical than you may have suspected, and more fun to tinker with! I am glad that Wube made the mechanics so interesting and I must praise our wonderful tinkerer community (especially *u/warbaque* and *u/flame_Sla* on Reddit) for sharing their explorations, tools, and guides so that this article became possible.
```

Comment:
```suggestion
After going through the nuts and bolts of nuclear power, hopefully it now seems easier to get into, more practical than you may have suspected, and more fun to tinker with! I am glad that Wube made the mechanics so interesting and I must praise our wonderful tinkerer community (especially *u/warbaque* and *u/flame_Sla* on Reddit) for sharing their explorations, tools, and guides so that this article became possible.
```